packages/sns/package.json

```diff
@@ -25,7 +25,7 @@
     "prepublishOnly": "npm run build:release"
   },
   "dependencies": {
-    "@lokalise/node-core": "^10.0.0",
+    "@lokalise/node-core": "^10.0.1",
```

Comment:
Should we update it also on AMQP and SQS?

Reply:
they are already up-to-date, as far as I can see

Reply:
Oh, sorry 🙇, thank youu 🙏

Reply:
np!